File: 1st/Jinseon/문자열겹쳐쓰기.py
def solution(my_string, overwrite_string, s):
    answer = my_string[:s] + overwrite_string
    if(len(my_string) > len(overwrite_string)):
        answer = my_string[:s] + overwrite_string + my_string[s+len(overwrite_string):]
    return answer

File: 3st/Juhee/문자열 겹쳐쓰기.java
class Solution {
    public String solution(String my_string, String overwrite_string, int s) {
        String answer = "";
        for(int i=0; i<s; i++){
            answer += my_string.charAt(i);
        }
        int j = 0;
        while(j<overwrite_string.length()){
            answer += overwrite_string.charAt(j);
            j++;
        }
        int k = j+s;
        while(k<my_string.length()){
            answer += my_string.charAt(k);
            k++;
        }
        return answer;
    }
}